### ISSUE TITLE
model_200.pth

### ISSUE THREAD

**Issue by normajeane-m** [sunhan1997/IndusGrasp#8]
Thank you for sharing the good research results in code. 

It seems that 'work_space/exp/model_200.pth' is not available right now. 
Could you share it if possible?

**Comment by sunhan1997** [comment]
Yes，the model_200.pth is a little big that cannot be upload, You can use the maskrcnn/train_bbox.py to train a new one., which is easy.

**Comment by pieris98** [comment]
Is it a similar model to the one you mentioned in #6 ? Can I use that model instead?

**Comment by sunhan1997** [comment]
> Is it a similar model to the one you mentioned in #6 ? Can I use that model instead?

Yes, you can try it. But I'm not so sure if there will be issues due to the version

**Comment by pieris98** [comment]
Hey @sunhan1997 ,  you're right the model was trained on an older version of PyTorch so a lot of source code has changed, this is a typical error when using models across PyTorch versions. I guess your recommended solution of training a MaskRCNN model is the only way to enable detection.

**Comment by sunhan1997** [comment]
you can also use other method to detect the object, such as yolo.




| |
孙晗
|
|
***@***.***
|
---- Replied Message ----
| From | Pieris ***@***.***> |
| Date | 10/10/2024 20:01 |
| To | ***@***.***> |
| Cc | ***@***.***>,
***@***.***> |
| Subject | Re: [sunhan1997/IndusGrasp] model_200.pth (Issue #8) |

Hey @sunhan1997 , you're right the model was trained on an older version of PyTorch so a lot of source code has changed, this is a typical error when using models across PyTorch versions. I guess your recommended solution of training a MaskRCNN model is the only way to enable detection.
